coursefinder/views.py

```diff
@@ -64,7 +64,7 @@ def course_finder_results(request):
     countries_query = request.GET.get('countries_query', None)
     url = "%s/search/institution-courses?subjects=%s" % (settings.SEARCHAPIHOST, subject_query)
     if institution_query != '':
-        url = url + "&institutions=%s" % (institution_query)
+        url = url + "&institutions=%s" % institution_query
```

Comment:
Not an issue, just a personal preference for f-strings. So would be url = f"{url}&institutions={institution_query}".

Reply:
I only removed the '(' here because the editor was complaining they weren't needed.
I have done a full refactor of this function, including this string concat, in my pagination branch (https://github.com/office-for-students/wagtail-CMS/pull/23).
Are you happy to leave as is in this one and have it fixed in the next PR?

Reply:
Yes no problem at all Matt. It was only an observation really.

Reply:
BTW, I wasn't really commenting on the missing parens. Just saying I would probably use f-strings for the formatting. 

```suggestion
        url = f"{url}&institutions={institution_query}"
```

But it's just my personal preference and the reason for that maybe that I enjoyed using string interpolation when coding in Perl :-)